docs/pages/features/networks.mdx

```diff
@@ -34,4 +34,14 @@ Users have the same Smart Wallet address across supported networks.
 
 ### tx.origin Behavior
 - `tx.origin` will be the bundler EOA address, not the Smart Wallet address
-- Consider this when integrating with existing contracts
\ No newline at end of file
+- Consider this when integrating with existing contracts
+
+## Adding Network Support
+
+For a network to be supported, it needs:
+
+- [Safe Singleton Factory](https://github.com/safe-global/safe-singleton-factory?tab=readme-ov-file#how-to-get-the-singleton-deployed-to-your-network) (CREATE2 deployer) deployed at `0x914d7Fec6aaC8cd542e72Bca78B30650d45643d7`
+- ERC-4337 Entrypoint v0.6 deployed at `0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789` 
+- Smart Wallet Factory deployed at `0x0BA5ED0c6AA8c49038F819E587E2633c4A9F428a`
+- ERC-4337 bundler meeting Coinbase's standards (available, latency, security)
```

Comment:
```suggestion
- ERC-4337 bundler meeting Coinbase's standards (liveness, latency, security)
```

The use of one adjective and two nouns was throwing me off here.  Would suggest a term like "liveness" or perhaps "availability", though that didn't feel particularly meaningful.

Reply:
Nice catch. This should be [availability](https://en.wikipedia.org/wiki/Reliability,_availability_and_serviceability), given its an eng term

Reply:
Fixed [availability](https://github.com/coinbase/smart-wallet-docs/pull/110/commits/f519c85ba05f1baa869ec664165fd8b83f665573)